source/brailleDisplayDrivers/papenmeier_serial.py

```diff
@@ -253,7 +253,13 @@ class InputGesture(braille.BrailleDisplayGesture):
 
 	source = BrailleDisplayDriver.name
 	
-	def __init__(self, keyindex, pressed, keys, driver):
+	def __init__(
+			self,
+			keyindex: Union[int, None],

```

Comment:
```suggestion
			keyindex: Optional[int],
```

Also the cases below